### PR TITLE
Feature/add okhttp client

### PR DESCRIPTION
**Pull request by beatarjn**

### CODE REVIEW

File: buns_store/src/main/java/org/example/ExampleClientController.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+@RestController
+public class ExampleClientController {
+
+    private OkHttpClient client = new OkHttpClient();
+
+    @GetMapping("/sendHello")
+    public String runHello() throws IOException {
+        Request request = new Request.Builder()
+                .url("http://localhost:8010/hello")
```
Comment (Andromedids) [inline buns_store/src/main/java/org/example/ExampleClientController.java:19]: 🎉 
Fantastic work!

Comment (Andromedids) [reply buns_store/src/main/java/org/example/ExampleClientController.java:19]: As a follow up (in another PR or in here, as you decide) I would encourage you to:
- extract the host's address ` to a field whose value would be actually taken from `properties` 
- extract its port also to properties

File: buns_store/src/main/java/org/example/ExampleClientController.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+import java.io.IOException;
+
+@RestController
+public class ExampleClientController {
+
+    private OkHttpClient client = new OkHttpClient();
+
+    @GetMapping("/sendHello")
+    public String runHello() throws IOException {
```
Comment (Andromedids) [inline buns_store/src/main/java/org/example/ExampleClientController.java:17]: let's add tests to that! 
We can use `WireMock` library for that for example:)

File: buns_store/build.gradle [hunk 3]
```diff
@@ -23,11 +23,13 @@ repositories {
 
 dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation group: 'com.squareup.okhttp3', name: 'okhttp', version: '4.12.0'
```
Comment (Andromedids) [inline buns_store/build.gradle:26]: let's standardize the approach in dependencies:
There are 2 options available:
1. Using the long version of dependencies like `implementation group: 'com.squareup.okhttp3', name: 'okhttp', version: '4.12.0`
2. Or using the shortened version like: `implementation 'com.squareup.okhttp3:okhttp:4.12.0`

Both are fine but let's use them in all places in dependencies ;) (currently `spring` related deps have the shortened way and this one has the longer way)

File: bakery_service/src/main/java/pl/rejmanbeata/bakery/controller/ExampleController.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ExampleController {
+
+
+    @GetMapping("/hello")
+    public ResponseEntity<String> getHello() {
+        return new ResponseEntity<>("Hello World!", HttpStatus.OK);
```
Comment (Andromedids) [inline bakery_service/src/main/java/pl/rejmanbeata/bakery/controller/ExampleController.java:15]: side note - we can static import `OK` and shortened this line

**Review by Andromedids — COMMENTED** [review]
Great work! Left some comments  but it's great job you did so far!

**Review by PaniOdProgramowania — COMMENTED** [review]
as I said in my previous review 🙈

**Pull request merged**